mozilla/fx-private-relay
# Warn users about exposing their emails with (B)CCs

# New feature description

This adds a warning message to forwarded emails for Premium users (because they can reply to emails) warning them that adding back CCs and BCCs will expose their real email address.

# How to test

~~**Question:** How do I deploy to `dev` before merging to `main`? I haven't been able to test this myself. (The procedure should be: send an email to a Premium account's alias, and verify that it shows the warning in the header and looks OK.)~~

I wrote instructions in the UI Engineering onboarding doc. This is what it looked like for me:

![Screen Shot 2021-12-06 at 18 32 52](https://user-images.githubusercontent.com/4251/144893822-609c1b71-5666-46c1-9a54-de1f05a49773.png)

# Checklist

- [x] l10n dependencies have been merged, if any. https://github.com/mozilla-l10n/fx-private-relay-l10n/pull/47 is merged.
